internal/database/users.go

```diff
@@ -802,6 +802,8 @@ type UsersListOptions struct {
 	Query string
 	// UserIDs specifies a list of user IDs to include.
 	UserIDs []int32
+	// Only show users inside this org
+	OrgId *int32
```

Comment:
I think it's better to make this an `int32`. We generally use `!= 0` checks in the database layer and its option structs. If you don't use a pointer here, the code in the graphql layer also becomes easier.

```suggestion
	OrgID int32
```